build.sbt

```diff
@@ -173,6 +175,19 @@ lazy val storage = (project in file("storage"))
     )
   )
 
+// TODO(scott): figure out a better way to include tests in this project
+lazy val storageDynamodb = (project in file("storage-dynamodb"))
+  .dependsOn(storage)
+  .settings (
+    name := "delta-storage-dynamodb",
+    commonSettings,
+    skipReleaseSettings,
+    // releaseSettings,
+    libraryDependencies ++= Seq(
+      "com.amazonaws" % "aws-java-sdk" % "1.7.4"
```

Comment:
what made you decide 1.7.4? i am curious.

Reply:
cc @mrk-its

Reply:
1.7.4 is the version we were using from the beginning of DynamoDBLogStore. AFAIR ir was version used by hadoop-aws 2.7 / spark 2.4 (at least on AWS EMR). And it still works with latest EMR versions, so we didn't change it. But we can of course try to bump it (I even tried to do this update some time ago, but immediately ran into sam problems with writing to S3, so it needs to be done very carefully)